addons/account_debit_note/wizard/account_debit_note_view.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="view_account_debit_note" model="ir.ui.view">
+            <field name="name">account.debit.note.form</field>
+            <field name="model">account.debit.note</field>
+            <field name="arch" type="xml">
+                <form string="Create Debit Note">
+                    <field name="move_type" invisible="1"/>
+                    <group>
+                         <group>
+                             <field name="reason"/>
+                             <field name="date" string="Debit Note Date"/>
+                             <field name="copy_lines" attrs="{'invisible': [('move_type', 'in', ['in_refund', 'out_refund'])]}"/>
```

Comment:
I believe here the domain should be "not in" because as it is now you can only use "copy_lines" when creating debit note from invoice, but it make more sence from credit notes (for me we can just remove the attrs and leave this option always)